save-demo/src/main/kotlin/com/saveourtool/save/demo/service/DownloadToolService.kt

```diff
@@ -90,7 +90,7 @@ class DownloadToolService(
 
     @Suppress("ReactiveStreamsUnusedPublisher")
```

Comment:
suggestion to remove this suppressing, it's should be applicable only for test purposes

Reply:
Oh
Idea highlights will make me suffer

Reply:
Will remove it

Reply:
Done

Reply:
I don't see that it was removed, also IDEA suggests it because there are unused publisher (I guess)

Reply:
Yep, DownloadToolService needs to be refactored